gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/auth/AppleAuth.java

```diff
@@ -51,11 +51,11 @@ public Promise<GdxFirebaseUser> signIn() {
             public void accept(final FuturePromise<GdxFirebaseUser> gdxFirebaseUserFuturePromise) {
                 ASAuthorizationAppleIDProvider appleIDProvider = new ASAuthorizationAppleIDProvider();
                 ASAuthorizationAppleIDRequest request = appleIDProvider.createRequest();
-                String nonce = hash256(UUID.randomUUID().toString());
+                String rawNonce = UUID.randomUUID().toString()
```

Comment:
`;` is missing at the end of the line. / I will fix it in master

Reply:
have no idea how it has passed the test pipelines. I need to verify github actions / could be that something is broken.

Reply:
oh I see .. it's still using travis. I need to migrate it to github actions soon. Plus it looks like travis pipeline is not triggered from the forked branch.